oci-unit-tests/cassandra_test_data/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM ubuntu:impish
+ARG CQLSH_VERSION=4.0.1
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get full-upgrade -y && \
 	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
 		git python3-pip build-essential python3-dev && \
-	git clone https://github.com/apache/cassandra.git && \
+	git clone -b cassandra-${CQLSH_VERSION} --single-branch https://github.com/apache/cassandra.git && \
```

Comment:
WDYT about using `--depth 1` (which implicitly means `--single-branch`) here to create a shallow clone (and save bandwidth/time)?

Reply:
+1! Fixed